shared/src/main/java/gov/hhs/cdc/trustedintermediary/external/hapi/HapiHelper.java

```diff
@@ -52,17 +96,419 @@ public static void addMetaTag(
         messageHeader.setMeta(meta);
     }
 
-    public static void setMessageTypeCoding(Bundle order, Coding coding) {
-        var messageHeader = findOrInitializeMessageHeader(order);
+    // MSH-3 - Sending Application
+    public static MessageHeader.MessageSourceComponent getMSH3MessageSourceComponent(
```

Comment:
This is only called by test code.  Let's remove it.  If you want this as a helper method to make testing easier, add it to (or create new) a helper class that is just inside the `test/groovy/` area.

Reply:
I'm willing to be pragmatic by leaving the methods that you think will be used as part of this story even though they aren't used in this specific PR.  But I'd like a follow-on PR at the end of this story to move the unused methods to the tests if they are unused by the real code.